app/routes/events/components/EventEditor/index.tsx

```diff
@@ -127,10 +145,91 @@ function EventEditor({
+                  imageGallery.map((e) => {
+                    return (
+                      <div key={e.key} className={styles.imageGalleryContainer}>
+                        <ConfirmModalWithParent
+                          title="Fjern fra galleri"
+                          message="Er du sikker på at du vil fjerne bildet fra bildegalleriet? (Bildet blir ikke slettet!)"
+                          onConfirm={() =>
+                            setSaveForUse(`${e.key}:${e.token}`, false)
+                          }
+                          closeOnConfirm
+                        >
+                          <button className={styles.closeButton}>
+                            <Icon name="close-circle" size={24} />
+                          </button>
+                        </ConfirmModalWithParent>
+                        <Image
+                          src={e.cover}
+                          placeholder={e.coverPlaceholder}
+                          alt={`${e.cover} bilde`}
+                          onClick={() => {
+                            change('cover', `${e.key}:${e.token}`);
+                            setShowImageGallery(false);
+                            setUseImageGallery(true);
+                            setImageGalleryUrl(e.cover);
+                          }}
+                          className={styles.imageGalleryEntry}
+                        />
+                      </div>
+                    );
+                  })}
+                {imageGallery.length === 0 && (
+                  <h2>
+                    Det finnes ingen bilder i bildegalleriet. Hvorfor ikke laste
+                    opp ett?
+                  </h2>
+                )}
+              </Flex>
+            </Flex>
+          </>
+        </Modal>
+
+        <Flex wrap alignItems="center" justifyContent="space-between">
+          <Flex alignItems="center" justifyContent="space-between">
+            <Button
+              className={styles.scannerButton}
+              onClick={() => setShowImageGallery(true)}
+            >
+              Velg bilde fra Bildegalleriet
+            </Button>
+            <Tooltip content="Velg et bilde som er lastet opp og markert for bruk fra tidligere arrangementer.">
+              <Icon
+                name="information-circle-outline"
+                size={20}
+                style={{
+                  cursor: 'pointer',
+                  marginLeft: '0.5em',
+                }}
+              />
+            </Tooltip>
+          </Flex>
+
+          <Tooltip content="Lagre bildet til bildegalleriet. Det kan da brukes i andre arrangementer.">
+            <Field
+              label="Lagre til bildegalleriet"
+              name="saveToImageGallery"
+              component={CheckBox.Field}
+              fieldClassName={styles.metaField}
+              className={styles.formField}
+              normalize={(v) => !!v}
+            />
+          </Tooltip>
```

Comment:
Regarding ABA-494, it would be nice if the tooltip did not wrap the entire field.